scripts/release/stage-release.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+set -ex
+
+# Stages a release by putting everything that should be packaged and released
+# into the ./deploy folder. This script should be run from the root of the
+# material2 repo.
+
+# Make sure you are not running `ng serve` or `ng build --watch` when running this.
+
+
+# Clear dist/ and deploy/ so that we guarantee there are no stale artifacts.
+rm -rf ./dist
+rm -rf ./deploy
+
+# For packaging for npm only, replace the TypeScript module format to commonjs
+TS_CONFIG_ORIGINAL=$(cat ./src/tsconfig.json)
+SYSTEM_FORMAT="\"module\": \"system\","
+COMMONJS_FORMAT="\"module\": \"commonjs\","
+TS_CONFIG_MODIFIED=${TS_CONFIG_ORIGINAL/${SYSTEM_FORMAT}/${COMMONJS_FORMAT}}
```

Comment:
Can you use `sed` here instead of some local variable?

```bash
sed -i .backup  "s/\(.*\"module\": \).*/\\1\"commonjs\",/" src/tsconfig.json
ng build
mv -f ${TSCONFIG_PATH}.backup ${TSCONFIG_PATH}
```